docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -0,0 +1,305 @@
+---
+---
+
+{% capture overview %}
+
+The goal of this codelab is for you to turn a simple Hello World node.js app 
+into a replicated application running on Kubernetes. We will show you how to 
+take code that you have developed on your machine, turn it into a Docker
+container image, and then run that image on [Minikube](/docs/getting-started-guides/minikube).
+
+Kubernetes is an open source project which can run on many different environments, 
+from laptops to high-availability multi-node clusters, from public clouds to 
+on-premise deployments, from virtual machines to bare metal. Minikube provides
+a simple way of running Kubernetes on your local workstation for free.
+
+{% endcapture %}
+
+{% capture objectives %}
+
+* Run a hello world Node application
+* Deploy the application to Minikube
+* View application logs
+* Update the application image
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* For OS X, you need [Homebrew](https://brew.sh) to install the xhyve
+driver.
+
+* Install Docker. On OS X, we recommend 
+[Docker for Mac](https://docs.docker.com/engine/installation/mac/).
+
+
+{% endcapture %}
+
+{% capture lessoncontent %}
+
+## Create a Minikube cluster
+
+This tutorial uses [Minikube](https://github.com/kubernetes/minikube) to 
```

Comment:
I don't like the assumption that the reader is using a Mac. I think we should find a way to make this tutorial OS-neutral. I think we should have a separate task/tutorial that shows the reader how to install Minikube on various operating systems. Then in this tutorial, it could be a prerequisite that Minikube is already installed.